### PR TITLE
dapr-cli: Add version 1.1.0

### PR DESCRIPTION
**Pull request by JadynWong**
Add Dapr CLI.

More info: https://github.com/dapr/cli

### CODE REVIEW

File: bucket/dapr-cli.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.0.0",
+    "description": "The Dapr CLI allows you to setup Dapr on your local dev machine or on a Kubernetes cluster, provides debugging support, launches and manages Dapr instances.",
```
Comment (earnestma) [inline bucket/dapr-cli.json:3]:
```suggestion
    "description": "Command-line tools for Dapr",
```

File: bucket/dapr-cli.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "homepage": "https://github.com/dapr/cli/",
```
Comment (earnestma) [inline bucket/dapr-cli.json:4]:
```suggestion
    "homepage": "https://github.com/dapr/cli",
```

**Review by earnestma — CHANGES_REQUESTED** [review]
Just minor nitpicks

**Comment by JadynWong** [comment]
> Just minor nitpicks

Thanks for the suggestion, already applied.

**Comment by JadynWong** [comment]
Hi @earnestma,
Can you help me to get this PR approved? 
Thank you very much!

**Comment by earnestma** [comment]
Would you mind updating the manifest to v1.0.1? https://github.com/dapr/cli/releases/tag/v1.0.1

**Comment by JadynWong** [comment]
> Would you mind updating the manifest to v1.0.1? https://github.com/dapr/cli/releases/tag/v1.0.1

Thanks.
It has been updated to v1.0.1.

**Review by chawyehsu — APPROVED** [review]

**Pull request merged**